app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -219,6 +223,15 @@ class QuestionPlayerActivityTest {
     testCoroutineDispatchers.unregisterIdlingResource()
   }
 
+  @Test
+  fun testActivity_createIntent_verifyScreenNameInIntent() {
+    val currentScreenName = QuestionPlayerActivity.createQuestionPlayerActivityIntent(
+      context, ArrayList(SKILL_ID_LIST), profileId
```

Comment:
```suggestion
      context, listOf(SKILL_ID_LIST), profileId
```

Nit--more Kotlin-idiomatic.

Reply:
done.

Reply:
Ah, it wasn't done but it appears that's because an ``ArrayList`` is required here.